cmd/talosctl/pkg/mgmt/helpers/wireguard.go

```diff
@@ -97,7 +97,7 @@ type WireguardConfigBundle struct {
 
 // PatchConfig generates config patch for a node and patches the configuration data.
 func (w *WireguardConfigBundle) PatchConfig(ip fmt.Stringer, cfg config.Provider) (config.Provider, error) {
-	config := cfg.Raw().(*v1alpha1.Config).DeepCopy()
+	config := cfg.RawV1Alpha1().(*v1alpha1.Config).DeepCopy()
```

Comment:
is it possible to make this more typed so that we can avoid type conversion in future PR's?

Reply:
yes, in the next PR, I need to move more things around to avoid import loop